api/v1/nodescan/service.go

```diff
@@ -270,6 +273,15 @@ func (s *serviceImpl) GetNodeVulnerabilities(_ context.Context, req *v1.GetNodeV
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	// Handle the new format of the request and scan node inventory additionally
+	if features.RHCOSNodeScanning.Enabled() && req.GetNodeInventory() != nil {
```

Comment:
If you go the path of removing feature flag, this line will turn into

```suggestion
	if req.GetNodeInventory() != nil {
```